### PR TITLE
feat: add new categories for Documenti su IO

### PR DESCRIPTION
**Pull request by mdegrego**
## Short description
Add a new category "Documenti su IO" and 6 sub-categories.

## List of changes proposed in this pull request
- Add a new category "Documenti su IO" and 6 sub-categories. Note that English and German labels will be provided later by Product & Design team.

## How to test
In app IO you need to click on '?' and create a new service request. Select "Documenti su IO" and one of its sub-categories.

### CODE REVIEW

**Comment by thisisjp** [comment]
@dhinterlechner help appreciated 🙏

**Comment by dhinterlechner** [comment]
> @dhinterlechner help appreciated 🙏

#850 😅

File: assistanceTools/zendesk.json [hunk 1]
```diff
@@ -40,6 +40,67 @@
...
+          "de-DE": "Dokumente in IO"
+        },
+        "zendeskSubCategories": {
+          "id": "29326690756369",
+          "subCategories": [
+            {
+              "value": "it_wallet_info",
+              "description": {
+                "it-IT": "Vorrei più informazioni",
+                "en-EN": "Vorrei più informazioni",
```
Comment (thisisjp) [inline assistanceTools/zendesk.json:57]:
```suggestion
                "en-EN": "I need more details",
```

File: assistanceTools/zendesk.json [hunk 2]
```diff
@@ -177,7 +238,7 @@
               "description": {
                 "it-IT": "Vorrei più informazioni",
                 "en-EN": "I'd like to have more information",
```
Comment (thisisjp) [inline assistanceTools/zendesk.json:240]:
```suggestion
                "en-EN": "I need more details",
```

File: assistanceTools/zendesk.json [hunk 3]
```diff
@@ -214,7 +275,7 @@
               "description": {
                 "it-IT": "Vorrei più informazioni",
                 "en-EN": "I would like to have more information",
```
Comment (thisisjp) [inline assistanceTools/zendesk.json:277]:
```suggestion
                "en-EN": "I need more details",
```

File: assistanceTools/zendesk.json [hunk 4]
```diff
@@ -40,6 +40,67 @@
...
+                "it-IT": "Non riesco ad aggiungere i documenti",
+                "en-EN": "Non riesco ad aggiungere i documenti",
+                "de-DE": "Ich kann keine Dokumente hinzufügen"
+              }
+            },
+            {
+              "value": "it_wallet_patente_guida",
+              "description": {
+                "it-IT": "Ho un problema con la Patente di Guida",
+                "en-EN": "Ho un problema con la Patente di Guida",
```
Comment (thisisjp) [inline assistanceTools/zendesk.json:81]:
```suggestion
                "en-EN": "I have an issue with the Driving Licence",
```

File: assistanceTools/zendesk.json [hunk 5]
```diff
@@ -40,6 +40,67 @@
...
+                "it-IT": "Ho un problema con la Tessera Sanitaria",
+                "en-EN": "Ho un problema con la Tessera Sanitaria",
+                "de-DE": "Ich habe ein Problem mit der Gesundheitskarte"
+              }
+            },
+            {
+              "value": "it_wallet_carta_disabilita",
+              "description": {
+                "it-IT": "Ho un problema con la Carta Europea della Disabilità",
+                "en-EN": "Ho un problema con la Carta Europea della Disabilità",
```
Comment (thisisjp) [inline assistanceTools/zendesk.json:97]:
```suggestion
                "en-EN": "I have an issue with the European Disability Card",
```

File: assistanceTools/zendesk.json [hunk 6]
```diff
@@ -40,6 +40,67 @@
...
+                "it-IT": "Ho un problema con la Patente di Guida",
+                "en-EN": "I have an issue with the Driving Licence",
+                "de-DE": "Ich habe ein Problem mit dem Führerschein"
+              }
+            },
+            {
+              "value": "it_wallet_tessera_sanitaria",
+              "description": {
+                "it-IT": "Ho un problema con la Tessera Sanitaria",
+                "en-EN": "Ho un problema con la Tessera Sanitaria",
```
Comment (thisisjp) [inline assistanceTools/zendesk.json:89]:
```suggestion
                "en-EN": "I have an issue with the Health Insurance Card",
```

**Review by LazyAfternoons — APPROVED** [review]
LGTM

**Pull request merged**